examples/undocumented/python/graphical/mclda.py

```diff
@@ -1,63 +1,57 @@
-from shogun import RealFeatures
 from shogun import MulticlassLabels
 from shogun import MCLDA
-from pylab import pcolor, contour, colorbar, connect, show, plot, axis
-
+import matplotlib.pyplot as plt
 import numpy as np
-import util
+from shogun import features
```

Comment:
actually I just looked through the changes here, the only functional change is replacing RealFeatures with Features.
The file above has no functional changes as far as I can see.